capstone/capdb/models.py

```diff
@@ -970,9 +1033,26 @@ def update_frontend_urls(cls, cite_strs, update_elasticsearch=True, batch_size=1
                     cls.reindex_cases(to_update)
 
     @classmethod
-    def reindex_cases(cls, cases):
+    def reindex_cases(cls, cases, prune_duplicates=False):
         from capapi.documents import CaseDocument  # avoid circular import
-        CaseDocument().update(cases)
+        if isinstance(cases, models.Model):
+            cases = [cases]
+
+        # only indexes non-duplicate cases
+        CaseDocument().update([case for case in cases if not case.duplicate])
```

Comment:
should use in_scope instead of duplicate?

Reply:
Yep.